server/src/app.ts

```diff
@@ -18,13 +19,23 @@ app.disable('x-powered-by');
 app.use((req, res, next) => {
   res.setHeader('X-XSS-Protection', '1; mode=block');
   res.setHeader('X-Content-Type-Options', 'nosniff');
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader('Access-Control-Allow-Methods', 'DELETE, POST, GET, OPTIONS');
```

Comment:
Do we have any methods with Delete and Options yet? Do we really need to set those?

Reply:
Without this options, I had CORS issue so I am planning to manually deploy one with options and another without options to see if I get a CORS issue.

Reply:
I added this to resolve the CORS issue. If we have to put this, I will delete `DELETE` method since we need other methods.